machines/arch_filelist.mk

```diff
@@ -0,0 +1,118 @@
+# This file contains a list of the *core* manycore architecture files that are
+# used in simulation. This has NOT been used in tapeout or any tapeout related
+# activities beyond simulation
```

Comment:
we shouldn't change file names for no good reason

Reply:
Maybe there is a good reason. I believe Bandhav is trying to solve a legitimate problem here, so maybe you should ask him why, if you don't understand

Reply:
@vb000 I don't understand why these filenames need to change?

Reply:
Previously file lists are just text files and now, they are make fragments. So, updated filenames so that people's text editors can detect make syntax.

Reply:
that depends on what text editor people use, doesn't it?